test/runtests.jl

```diff
@@ -3,40 +3,46 @@ include("includes.jl")
 # Code Quality Tests
 import Aqua
 Aqua.test_undefined_exports(PowerSimulations)
-Aqua.test_ambiguities(PowerSimulations)
-#Aqua.test_stale_deps(PowerSimulations)
+#Aqua.test_ambiguities(PowerSimulations)
```

Comment:
I feel like at least this one is worth trying to fix before merging this into `psy5`

Reply:
It's a good aspiration but if it isn't fixed by Friday EI testing we need to leave it.